README.md

```diff
@@ -1,6 +1,6 @@
-# oci-proxy
+# registry.k8s.io
 
-This project is currently an experiment, for more see [cmd/archeio](./cmd/archeio/README.md)
+The production redirect domain for Kubernetes artifacts, for more see [cmd/archeio](./cmd/archeio/README.md)
```

Comment:
```suggestion
The production redirector service for container images of the Kubernetes Project, for more see [cmd/archeio](./cmd/archeio/README.md)
```
we only serve container images at the moment.

Reply:
I updated the description in e0679f7. 
I've gone with 
> The production redirector service for Kubernetes container image artifacts

I think it's more concise.